apm-sniffer/apm-sdk-plugin/spring-plugins/mvc-annotation-commons/src/main/java/org/apache/skywalking/apm/plugin/spring/mvc/commons/interceptor/AbstractMethodInterceptor.java

```diff
@@ -208,4 +222,31 @@ private void collectHttpParam(HttpServletRequest request, AbstractSpan span) {
             Tags.HTTP.PARAMS.set(span, tagValue);
         }
     }
+
+    private boolean shouldCollectHeader() {
+        List<String> includeHeaders = SpringMVCPluginConfig.Plugin.Http.INCLUDE_HTTP_HEADERS;
+        return includeHeaders != null && includeHeaders.size() > 0 ;
+    }
+
+    private void collectHttpHeaders(HttpServletRequest request, AbstractSpan span) {
+        final Enumeration<String> headerNames =  request.getHeaderNames();
+        if (headerNames == null) {
+            return;
+        }
+        final List<String> headersList = new LinkedList<>();
+        Collections.list(headerNames).stream().forEach(headerName -> {
+            Enumeration<String> headerValues = request.getHeaders(headerName);
+            String headerValue = Collections.list(headerValues).toString();
+            if (shouldCollectHeader() && SpringMVCPluginConfig.Plugin.Http.INCLUDE_HTTP_HEADERS.contains(headerName.toLowerCase())) {
```

Comment:
`SpringMVCPluginConfig.Plugin.Http.INCLUDE_HTTP_HEADERS not empty` should be at the first line of the method, to provide `return` fast.